lib/phoenix_html/format.ex

```diff
@@ -39,6 +39,7 @@ defmodule Phoenix.HTML.Format do
 
     string
     |> maybe_html_escape(escape?)
+    |> String.replace("\r", "")
```

Comment:
Removing the `\r` is the cleanest way I think.
Could also do add split patterns:

```ex
|> String.split(["\n\n", "\r\n\r\n"], trim: true)
```

But would have to do this in multiple places.

Reply:
It feels semantically wrong though. I would prefer to add split patterns! :+1:

Reply:
> It feels semantically wrong though. I would prefer to add split patterns!

Done! Let me know what you think :)

Reply:
Done! let me know what you think :)